contributing/ci-bio-formats.txt

```diff
@@ -126,27 +132,56 @@ The branch for the 5.1.x series of Bio-Formats is develop.
 
 	:jenkinsjob:`BIOFORMATS-5.1-merge-full-repository`
 
-		This job runs automated tests against the full repository on squig
+		This job runs automated tests against the full repository on Linux
 
-		#. Checks out the merge/develop/latest branch of the
+		#. Checks out the develop/merge/daily branch of the
 		   snoopycrimecop fork of bioformats.git_
 		#. Runs automated tests against :file:`/ome/data_repo/from_skyking/`
 
+	:jenkinsjob:`BIOFORMATS-5.1-merge-full-repository-win`
+
+		This job runs automated tests against the full repository on Windows
+
+		#. Checks out the develop/merge/daily branch of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\from_skyking`
```

Comment:
Will fix the escaping. The name is actually right, `/ome/data_repo/` is mounted as SMB share `ome-data-repo` for the Windows testing.